commands/info.js

```diff
@@ -0,0 +1,89 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+
+function configVarNamesFromValue (config, value) {
+  const sortBy = require('lodash.sortby')
+
+  let keys = []
+  for (let key of Object.keys(config)) {
+    if (config[key] === value) keys.push(key)
+  }
+  return sortBy(keys, k => k !== 'DATABASE_URL', 'name')
+}
+
+function databaseNameFromUrl (uri, config) {
+  const invert = require('lodash.invert')
+  const url = require('url')
+
+  delete config.DATABASE_URL
```

Comment:
I know it doesn't matter to your only caller, but it seems like a future footgun to mutate your argument here. Can you do something like `let conf = Object.assign({}, config)` and then use `conf`?